ocatool/Cargo.toml

```
@@ -14,9 +14,10 @@ name = "ocatool"
 path = "src/main.rs"
 
 [dependencies]
+dirs = "1.0.5"
```

there's a newer version [dirs - crates.io](https://crates.io/crates/dirs)
```suggestion
dirs = "5.0.1"
```